Library/Homebrew/os/mac/ruby_relocate.rb

```diff
@@ -0,0 +1,32 @@
+require "vendor/macho/macho"
+
+module RubyRelocate
+  def change_dylib_id(id, file)
+    puts "Changing dylib ID of #{file}\n  from #{file.dylib_id}\n    to #{id}" if ARGV.debug?
+    MachO::Tools.change_dylib_id(file, id)
+  end
+
+  def change_install_name(old, new, file)
+    puts "Changing install name in #{file}\n  from #{old}\n    to #{new}" if ARGV.debug?
+    MachO::Tools.change_install_name(file, old, new)
+  end
+
+  # only here until old relocation code is removed
+  def require_install_name_tool?
+    true
```

Comment:
Yeah, I misunderstood the semantics of `require_install_name_tool?`. I'll fix this in a moment.